solidity/deploy/07_disable_redemptions.ts

```diff
@@ -0,0 +1,40 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const { deployments, ethers, getNamedAccounts } = hre
+  const { execute, read } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  deployments.log("disabling redemptions in the Bridge")
+
+  // To disable redemptions, we need to make the redemption dust threshold
+  // extremely high. The redemption dust threshold value is determined by the
+  // redemptionDustThreshold governance parameter. This parameter is uint64.
+  // The maximum value for the uint64 is 2^64-1 = 18446744073709551615.
+  const redemptionDustThreshold = ethers.BigNumber.from("18446744073709551615")
```

Comment:
I do not think this is enough, `2^64-1` in e18 precision is just 18 TBTC. It is not a tiny amount but it's also not that high amount. We need one more thing: what about setting `redemptionTimeout` to the max possible value of `2^32-1 = 4294967295` which is 136 years?

Reply:
I've just merged https://github.com/keep-network/tbtc-v2/pull/446 to unblock this work, but if we decide to set `redemptionTimeout` to the max value allowed by the type, I think we should do the same for `fraudChallengeDefeatTimeout` in `06_disable_fraud_challenges.ts`, for consistency.

Reply:
See d372066d and f236d3cc